Components/Optimizers/AdaptiveStochasticLBFGS/elxAdaptiveStochasticLBFGS.hxx

```diff
@@ -236,11 +236,11 @@ void AdaptiveStochasticLBFGS<TElastix>
   this->GetConfiguration()->ReadParameter( this->m_AutomaticLBFGSStepsizeEstimation,
     "AutomaticLBFGSstepsizeEstimation", this->GetComponentLabel(), level, 0 );
 
-  /** Set the GradientMagnitudeTolerance *
+  /** Set the GradientMagnitudeTolerance */
   double gradientMagnitudeTolerance = 0.000001;
   this->m_Configuration->ReadParameter( gradientMagnitudeTolerance,
     "GradientMagnitudeTolerance", this->GetComponentLabel(), level, 0 );
-  this->SetGradientMagnitudeTolerance( gradientMagnitudeTolerance );
+  // this->SetGradientMagnitudeTolerance( gradientMagnitudeTolerance );
 
```

Comment:
why is this one commented out?

Reply:
`AdaptiveStochasticVarianceReducedGradient::SetGradientMagnitudeTolerance` is not implemented

Reply:
Thanks @haampie Now I see, the class does have a `m_GradientMagnitudeTolerance` data member, but it isn't being used. Andf indeed, it does not have a `Set` function anymore. So my original suggestion to restore the `ReadParameter` was wrong! The whole section that was commented out (accidentally, I assumed) can just go away!